### PR TITLE
PR for issue #27

### LINKED ISSUE #27: Implement CO-OP pong game

### PR DESCRIPTION
**Pull request by tnaceur**
This pull request will resolve #27

### CODE REVIEW

File: frontend/pages/GamePage.js [hunk 1]
```diff
@@ -7,7 +9,11 @@ export default class GamePage extends HTMLElement {
     const template = document.getElementById("game-template");
     const component = template.content.cloneNode(true);
     this.appendChild(component);
+    const canvas = document.getElementById("canvas-pong");
+    const ctx = canvas.getContext('2d');
+    runGame(canvas, ctx);
   }
 }
 
+
```
Comment (zwzone) [inline frontend/pages/GamePage.js:18]: an empty line here

**Review by zwzone — APPROVED** [review]

File: frontend/pong/pong.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    canvas;
+    ctx.fillStyle = "whitesmoke";
+    ctx.fillRect(this.positionX, this.positionY, this.sizeX, this.sizeY);
+  }
+
+  Center() {
+    return [this.positionX + this.sizeX / 2, this.positionY + this.sizeY / 2];
+  }
+}
+
```
Comment (reogine) [inline frontend/pong/pong.js:60]: Added unused function

File: frontend/pong/pong.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
+export function runGame(canvas, ctx) {
+  canvas.width = 1920;
+  canvas.height = 1080;
+  const ball = new Ball([10, 10], [canvas.width / 2, canvas.height / 2], 20);
```
Comment (reogine) [inline frontend/pong/pong.js:4]: the ball should change the direction randomly in every start

File: frontend/pong/pong.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+window.addEventListener("keydown", function (e) {
+  KeyPressed[e.keyCode] = true;
+});
+
+window.addEventListener("keyup", function (e) {
+  KeyPressed[e.keyCode] = false;
+});
+
+function BallPaddleCollision(ball, paddle) {
```
Comment (reogine) [inline frontend/pong/pong.js:98]: when the ball hit the upper side or down side of the paddle a glitch appeares

File: frontend/css/pong.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#canvas-pong {
+    width: 80%;
+    height: 50%;
+    background: url("../public/background.jpg");
+    background-position: center;
+    background-size: cover;
+    border-radius: 8px;
+}
```
Comment (reogine) [inline frontend/css/pong.css:8]: EOF shoud be a newline

**Review by reogine — APPROVED** [review]

**Pull request merged**